workers/unity/Assets/Playground/Scripts/Bootstrap.cs

```diff
@@ -64,12 +64,20 @@ public void Awake()
                 var workerId =
                     CommandLineUtility.GetCommandLineValue(commandLineArgs, RuntimeConfigNames.WorkerId,
                         string.Empty);
+                var useDynamicWorkerId = CommandLineUtility.GetCommandLineValue(commandLineArgs,
+                    RuntimeConfigNames.UseDynamicWorkerId,
+                    false);
 
                 // because the launcher does not pass in the worker type as an argument
                 var worker = workerType.Equals(string.Empty)
                     ? WorkerRegistry.CreateWorker<UnityClient>($"{workerType}-{Guid.NewGuid()}", new Vector3(0, 0, 0))
                     : WorkerRegistry.CreateWorker(workerType, workerId, new Vector3(0, 0, 0));
 
+                if (useDynamicWorkerId)
+                {
+                    worker.UseDynamicId = true;
```

Comment:
`worker.UseDynamicId = useDynamicWorkerId`? :)
Or just collapse the assignment to the lookup above.

Reply:
I want the `useDynamicWorkerId` to override only if it's true. If it's false, it should not make `worker.UseDynamicId` to false. I should rename `use` to `forceUse` perhaps and add a comment around the if block